samples/other/dotnet/csharp-ooproc/CSharpIsolatedSamples/ChatBotIsolated.cs

```diff
@@ -26,7 +26,7 @@ public class CreateRequest
     {
         var responseJson = new { chatId };
 
-        using (StreamReader reader = new StreamReader(req.Body))
+        using StreamReader reader = new StreamReader(req.Body);
         {
```

Comment:
You can move the open and closing curly braces now.

https://www.syncfusion.com/blogs/post/csharp-8-0-using-declaration.aspx